packages/bbui/src/Modal/ModalContent.svelte

```diff
@@ -18,6 +18,7 @@
   export let onCancel = undefined
   export let disabled = false
   export let showDivider = true
+  export let cancelOnClose = false
```

Comment:
Would we be able to just supply the `onCancel` function?
Or what problem is preventing it? If that doesn't work for some reason we should probably fix that rather than introduce the new field. I think we would always want to run the `onCancel` logic if the modal is exited by the `Cancel` or the `x` button

Reply:
I was worried about changing the behaviour in other modals, the idea of the flag was to make it optional, but I'm definitely happy to remove the flag if you think that's the way to go?